Packages/manifest.json

```diff
@@ -9,7 +9,7 @@
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.learn.iet-framework": "2.2.1",
     "com.unity.memoryprofiler": "0.5.0-preview.1",
-    "com.unity.netcode.gameobjects": "1.0.0-pre.9",
+    "com.unity.netcode.gameobjects": "1.0.0-pre.10",
```

Comment:
wait, there's no `pre.10` yet 😛

Reply:
yep, that's why I didn't enable auto merge yet :P I'll mark this PR as on hold for now also just in case